micrometer-commons/src/main/java/io/micrometer/common/Event.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.common;
+
+/**
+ * Represents an event used for documenting instrumentation.
+ *
+ * @author Marcin Grzejszczak
+ * @since 1.10.0
+ */
+public interface Event {
+
+    /**
+     * Returns event name. Use can use {@code %s} to represent dynamic entries that should
```

Comment:
Isn't this true only for the contextualName?
Do we want to call out that the name should have low cardinality while the contextualName can have high cardinality?

Reply:
Copy paste :see_no_evil: